src/content/reference/cli.mdx

```diff
@@ -59,7 +59,7 @@ $ okteto build [service...]
 ```
 
 If your [okteto context](/docs/reference/cli/#context) points to an Okteto URL, images are built using the [Okteto Build Service](/docs/cloud/build/).
-Otherwise, images are built using your local Docker daemon.
+Otherwise, images are built using your local Docker daemon. If you want to use your own BuildKit instance, set the variable `BUILDKIT_HOST` to point to your Buildkit server address.
```

Comment:
```suggestion
Otherwise, images are built using your local Docker daemon. If you want to use another BuildKit instance, set the variable `BUILDKIT_HOST` to point to its address, using the format `HOST:PORT`. 
```